R/default_transition_functions.R

```diff
@@ -15,19 +13,17 @@
 #' expo_fun(.5, NULL, .1, .3)
 #'
 #' @export
-expo_fun <- function(x, y, a, b) ifelse(x > 0, a * x^b, 0)
```

Comment:
changes in this file are just removing no-longer-needed placeholder arguments for predictors